src/main/java/seedu/address/storage/JsonAdaptedBook.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.book.Book;
+public class JsonAdaptedBook {
```

Comment:
Add a javadoc here and put space between import and class

Reply:
Oh yeah, I totally forget that one. Maybe I pass this javadoc job to the next person? So that we have equally shared documentation.